### DISCUSSION TITLE
最新内测版微信3.7.08不能用了

### DISCUSSION THREAD

**Discussion by Tree-New-Beee** [huiyadanli/RevokeMsgPatcher#328] [General]
最新内测版微信3.7.08不能用了

**Comment by huiyadanli** [comment]
发个安装包

**Reply by Tree-New-Beee** [reply]
https://www.123pan.com/s/A6cA-k9AJh

**Comment by Tree-New-Beee** [comment]
https://www.123pan.com/s/A6cA-k9AJh-------- 原始邮件 --------发件人： 辉鸭蛋 ***@***.***>日期： 2022年5月14日周六 12:33收件人： huiyadanli/RevokeMsgPatcher ***@***.***>抄送： Tree-New-Beee ***@***.***>, Author ***@***.***>主    题： Re: [huiyadanli/RevokeMsgPatcher] 最新内测版微信3.7.08不能用了 (Discussion #328)

发个安装包

**Reply by shuanggen688** [reply]
好像3.0版本以上都不会显示已撤回.能否添加上去?现在最新版本是3.6左右的了.不显示已撤回有点不太习惯

**Comment by huiyadanli** [comment]
#329  提供一个新的补丁程序，试试

**Reply by shuanggen688** [reply]
好像3.0版本以上都不会显示已撤回.能否添加上去?现在最新版本是3.6左右的了.不显示已撤回有点不太习惯

**Comment by CLAY-CMD** [comment]
这是来自QQ邮箱的假期自动回复邮件。
 
您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。
